src/proto/beats.proto

```diff
@@ -11,17 +11,36 @@ enum Source {
   USER = 2;
 }
 
-enum Type {
-  BEAT = 0;
-  BAR = 1;
-  PHRASE = 2;
+enum SectionType {
+  INTRO = 0;
+  INNER = 1;
+  OUTRO = 2;
 }
 
 message TimeSignature {
   optional int32 beats_per_bar = 1 [ default = 4];
   optional int32 note_value = 2 [ default = 4 ];
 }
 
+message TimeSignatureMarker {
+  optional int32 beat_index = 1 [ default = 0 ];
+  optional TimeSignature time_signature = 2;
+}
+
+message PhraseMarker {
+  optional int32 beat_index = 1 [ default = 0 ];
+}
+
+message Section {
+  optional string name = 1 [ default = "UNNAMED"];
+  optional SectionType type = 2 [ default = INNER ];
+}
+
+message SectionMarker {
+  optional int32 beat_index = 1 [default = 0];
+  optional Section section = 2;
+}
```

Comment:
Why are these separate messages? Also, aren't we adding a color for sections?

Reply:
These can be merged into only the SectionMarker. And we do have Color in the latest commits.

Reply:
Or maybe I should remove or comment Sections and Phrases from the protobuf since they won't be implemented in this PR anyways.

Reply:
Okay, let's comment those out for now. We can uncomment them when we start to use them.